lambdas/ping.js

```diff
@@ -0,0 +1,7 @@
+if (process.env.NODE_ENV === 'development') {
+  require('dotenv').config();
+}
+
+module.exports = (req, res) => {
+  res.send({ pong: true });
```

Comment:
I'm not enterily sure (my first time using now too) but I think node's functons have a different interface than express:
- When using `micro`, you import `send` and do `send(res, 200, { pong: true })`.
- When using the `res` directly, it should be `res.end(JSON.stringify({pong: true})`.